Revit_Core_Engine/Query/ElementType.cs

```diff
@@ -262,6 +282,17 @@ public static ElementType ElementType(this Document document, string familyName,
 
         /***************************************************/
 
+        private static void LuminaireFamilyAndTypeNames(this Luminaire luminaire, out string familyName, out string familyTypeName)
```

Comment:
You could call it simply `FamilyAndTypeNames`

Reply:
Method removed to align with other Types